frontend/app/routes/$lang+/_public+/apply+/$id+/personal-information.tsx

```diff
@@ -116,7 +117,8 @@ export async function action({ context: { session }, params, request }: ActionFu
         .string()
         .trim()
         .optional()
-        .transform((val) => val?.toUpperCase()),
+        .transform((val) => val?.toUpperCase())
```

Comment:
This transform is not needed. `formatPostalCode` should take care of formating and changing the case to upper case.